sklearn/tree/tree.py

```diff
@@ -512,9 +512,9 @@ def decision_path(self, X, check_input=True):
     def feature_importances_(self):
         """Return the feature importances.
 
-        The importance of a feature is computed as the (normalized) total
-        reduction of the criterion brought by that feature.
-        It is also known as the Gini importance.
+        We measure the importance of a feature by calculating
```

Comment:
This is wrong. The docstring is correct here. The `feature_importances_` is computed using the Gini importance.
What needs to be changed is the User Guide in which we need to change the occurrence feature importance by additional information mentioning that this is the Gini importance.

And as @amueller mentioned, whenever possible we should change our example or User Guide to use the feature importance based on random permutation instead.